### PR TITLE
Adding FS TC-83573992

### PR DESCRIPTION
**Pull request by julpark-rh**
Automation - T2 - CEPH-83573992 - List the nfs export details within nfs cluster in detail with --detailed option and note all the details are proper

https://polarion.engineering.redhat.com/polarion/redirect/project/CEPH/workitem?id=CEPH-83573992

pre-requisites:
1. nfs cluster ready
2. export the nfs cluster
Test operation:
1. check if input and --detailed value match
2. check if nfs clsuter info using its cluster_id

Log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1MOSS5/

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/nfs/nfs_info_cluster_id_and_ls_export_detailed.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        cmd_nfs_export = (
+            f"ceph nfs export create cephfs {cluster_id} {bind} {fs_name} --path={path}"
+        )
+
+        client1.exec_command(sudo=True, cmd=cmd_nfs_export)
+
+        res, ec = client1.exec_command(
+            sudo=True, cmd=f"ceph nfs export ls {cluster_id} --detailed --format json"
+        )
+        results = json.loads(res)
```
Comment (AmarnatReddy) [inline tests/cephfs/nfs/nfs_info_cluster_id_and_ls_export_detailed.py:75]: can we log results for quick reference in future

**Review by hkadam134 — COMMENTED** [review]
Looks Good, As Amar said, ensure the output of "--detailed" command and "cluster-info"  are captured and displayed.

File: tests/cephfs/nfs/nfs_info_cluster_id_and_ls_export_detailed.py (hunk 1 above)
Comment (julpark-rh) [reply tests/cephfs/nfs/nfs_info_cluster_id_and_ls_export_detailed.py:75]: @AmarnatReddy I have logged the results

Comment (hkadam134) [reply tests/cephfs/nfs/nfs_info_cluster_id_and_ls_export_detailed.py:75]: @julpark-rh - Can you add the logs link

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**